packages/admin-ui/src/calendar/utils.ts

```diff
@@ -20,6 +22,36 @@ export function useWeekDays(weekStart: number): WeekDay[] {
   })
 }
 
+export function createDate(date?: DateObject): Date {
+  if (!date) {
+    return new Date()
+  }
+
+  const { year, month, day } = date
+
+  invariant(year >= 0, 'The year must be greater then 0')
+  invariant(month >= 0 && month <= 11, 'The month must be between from 0 to 11')
```

Comment:
```suggestion
  invariant(month >= 0 && month <= 11, 'The month must be between 0 and 11')
```

Reply:
Would it be possible to change the range to 1 to 12? I'm just wondering if this could make more sense, since do not work with 0 month 🤔 Maybe I more realistic number could help here.

Reply:
I know we can work with month "0" in our code, but it is more a suggestion imagining that could make easier to understand de range or the error message.

Reply:
I thought about when I was implementing this. The tradeoff in my head was: For someone that is not used to JS dates, the 1 to 12 is more intuitive. For someone that used to the date object, the 0 to 11 is already a pattern. I choose to keep the native behavior to be more inclusive with every JS developer, at any timezone, that speaks any language. I can be wrong though... Does it make sense?

Reply:
We could stick with this approach/decision and pay attention if t will be any comments/requests about it 😉